RobotFramework_Testsuites/Config/CConfig.py

```diff
@@ -414,9 +416,20 @@ def updateCfg(sUpdateCfgFile):
         except:
             pass
         
-        BuiltIn().set_global_variable("${CONFIG}", oTmpJsonCfgData)
+        bDotdict = False
+        dotdictObj = CConfig.CJsonDotDict()
+        try:
+            jsonDotdict = dotdictObj.dotdictConvert(oTmpJsonCfgData)
+            bDotdict = True
+        except:
```

Comment:
Hello Thomas,

For your question "And why is this an info? We should exit here. Why should we process futher files when we have an error here?"
At the beginning during supporting BIST:RF or TA Framework, we recognized that some parameters in their configuration files contain "." in a parameter name. It makes dotdict converting process not successful, then we defined skip dotdict converting in this case.
I also mention this in our document: 
Note: In case a parameter name contains a ”.”, then it is not possible to use dotdict but the traditional way
${dict}[abc][def] is still working.

Thank you,
SOn

Reply:
Hi Son,
O.K. I got the point. Accepted...
Thank you for the explanation,
  Thomas